plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -436,11 +448,16 @@ func ListRegisteredFitPredicates() []string {
 	schedulerFactoryMutex.Lock()
 	defer schedulerFactoryMutex.Unlock()
 
-	names := []string{}
+	names := sets.NewString()
+
 	for name := range fitPredicateMap {
-		names = append(names, name)
+		names.Insert(name)
 	}
-	return names
+	for name := range mandatoryFitPredicateMap {
+		names.Insert(name)
+	}
+
+	return names.List()
```

Comment:
I believe it's fine, but you need to notice here, that the order of predicates will be randomized here.